internal/logger/logger.go

```diff
@@ -32,8 +35,7 @@ const GCSFuseInBackgroundMode string = "GCSFUSE_IN_BACKGROUND_MODE"
 
 var (
 	defaultLoggerFactory *loggerFactory
```

Comment:
are we supporting different logging format also?

Reply:
We should support log-file and log-format also in the new config file. Half config here and half config there is not ideal.

Reply:
Raised a separate PR for these changes: https://github.com/GoogleCloudPlatform/gcsfuse/pull/1362